pkgs/build-support/build-fhs-userenv-bubblewrap/default.nix

```diff
@@ -126,6 +139,13 @@ let
       done
     fi
 
+    declare -a graphics_share
+    for dir in /run/opengl-driver/share/*; do
+      if [[ -d "$dir" ]]; then
+        graphics_share+=(--bind "$dir" "/usr/share/$(basename $dir)")
```

Comment:
Are they available at both `/run/opengl-driver` and `/usr` in the FHS?

Reply:
yes, /usr/share is the default though, so it integrates nicer with other conventions